Sign-up.dart

```diff
@@ -0,0 +1,119 @@
+import 'dart:io';
+import 'dart:convert';
+import 'Welcome.dart';
+import 'home.dart';
+// ignore: unused_import
+import 'package:crypt/crypt.dart';
+
+class Sing_up {

```

Comment:
Underscore is usually used in naming vars or constants but not classes.
```suggestion
class SignUp {
```